ovs/lib/albanode.py

```diff
@@ -290,13 +290,19 @@ def fill_slots(node_guid, osd_information, metadata=None):
                 # Only filling is required
                 AlbaNodeController._fill_slot(node, osd_info['slot_id'], dict((key, osd_info[key]) for key in node.node_metadata['fill_metadata']))
             elif node.node_metadata['fill_add'] is True:
+                AlbaNodeController._logger.info('Osd info on create')
+                AlbaNodeController._logger.info(osd_info)
                 # Fill the slot
                 created_osds = AlbaNodeController._fill_slot(node, osd_info['slot_id'], dict((key, osd_info[key]) for key in node.node_metadata['fill_add_metadata']))
                 # And add/claim the OSD
                 if node.type == AlbaNode.NODE_TYPES.S3:
                     # The S3 manager returns the information about the osd when filling it
+                    AlbaNodeController._logger.info('Created osds')
+                    AlbaNodeController._logger.info(created_osds)
                     for created_osd_info in created_osds:
                         osd_info.update(created_osd_info)  # Add additional information about the osd
+                        AlbaNodeController._logger.info('Osd info add osds')
```

Comment:
this way, only logging is provided when an s3-type osd is created. 
If the logging is done in the `albacontroller.add_osds`, so should this for the creation of s3 osds. 
Only logging for s3 creation makes no sense imho. You removed the logging in `_fill_slots` -> logging for all osds makes more sense